app/models/applicant.rb

```diff
@@ -12,45 +12,11 @@ class Applicant < ApplicationRecord
   has_many :bank_providers
   has_many :bank_errors
 
-  validates :first_name, :last_name, :date_of_birth, :national_insurance_number, presence: true
-
-  validate :validate_date_of_birth, :validate_national_insurance_number
-
-  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }, allow_nil: true
-
   def email_address
     email
   end
 
   def full_name
     "#{first_name} #{last_name}".strip
   end
-
-  def dob
-    {
-      dob_year: date_of_birth.strftime('%Y'),
-      dob_month: date_of_birth.strftime('%m'),
-      dob_day: date_of_birth.strftime('%d')
-    }
-  end
-
-  private
-
-  def validate_date_of_birth
-    errors.add(:date_of_birth, :invalid) if date_of_birth.present? && (date_of_birth < Date.new(1900, 0o1, 0o1) || date_of_birth > Date.today)
```

Comment:
we don't need this anymore?

Reply:
That's now done solely by `BasicDetailsForm` lines 15-21.

Reply:
Ah ok I didn't realise it was duplicated